### PR TITLE
Text fixes in Getting Started and Introduction pages.

### PR DESCRIPTION
**Pull request by adiatzmony**

### CODE REVIEW

File: getting_started.rst [hunk 1]
```diff
@@ -1,170 +1,160 @@
...
 ----------------------------
 
 First, let's create a folder for our project:
 
 .. code-block:: bash
 
-    $ mkdir mytimer
-    $ cd mytimer
-
+   $ mkdir mytimer
```
Comment (memsharded) [inline getting_started.rst]: Please do not change indentation of code-blocks. Not necessary, and introduce a lot of noise for the review. Thanks!

File: getting_started.rst [hunk 2]
```diff
@@ -1,170 +1,160 @@
...
     .. code-block:: bash
     
        $ git clone https://github.com/memsharded/example-poco-timer.git mytimer
+       
+       
+Next, create the following source files inside this folder:
+
 
-Create the following source files inside that folder:
+**timer.cpp**
```
Comment (memsharded) [inline getting_started.rst]: Leave the file name as :caption: as it was originally.
Also, as above, do not reindent the source code inside code-blocks.

File: getting_started.rst [hunk 3]
```diff
@@ -181,26 +171,23 @@ Now, you are ready to build and run your project:
...
 
-The retrieved packages have been installed to your local user cache (typically ``.conan/data``), so
-they can be reused from there in other projects, and allow to clean your current project and keep
-working even without network connection. To search packages in the local cache you can do:
+The retrieved packages are installed on your local user cache (typically ``.conan/data``), and can be reused in other projects. This allows to keep you current project clean and to continue working even without network connection. Search packages in the local cache using:
 
 .. code-block:: bash
 
-    $ conan search
+    $ conan search
```
Comment (memsharded) [inline getting_started.rst]: No trailing whitespace necessary

File: getting_started.rst [hunk 4]
```diff
@@ -235,48 +220,31 @@ You can generate a graph of your dependencies, in dot or html formats:
...
 
-In this example we have built our project using the default configuration detected by conan, this
-configuration is known as the :ref:`default profile<default_profile>`.
-
-The first time you run the **conan** command, your settings are detected (compiler,
-architecture...) automatically and stored as default in a profile. You can change your those
-settings by editing ``~/.conan/profiles/default`` or create new profiles with the desired
-configuration.
-
-For example, if we have a profile with a gcc configutarion for 32 bits in a profile called
```
Comment (memsharded) [inline getting_started.rst]: You are removing some valid contents that were added recently, in order to solve the conflicts.
Don't worry about this one, I can add back those contents later.

**Comment by danimtb** [comment]
Just to take into account for future contributions: https://github.com/conan-io/docs#style-guidelines

**Comment by danimtb** [comment]
Hi @adiatzmony!

We have finally included your work in an updated PR.
Thank you so much for your contribution and new revisions are always welcomed!

Cheers,

**Pull request closed without merging**